documentation/source/install/install_usecases.rst

```diff
@@ -201,3 +201,157 @@ The above command can be invoked using the *watch* command by providing a desire
+
+::
+  
+    kubectl apply -f openebs-operator.yaml
+
+Add the OpenEBS storage classes using the following command, that can then be used by developers to map a suitable storage profile for their applications in their respective persistent volume claims.    
+
+::
+  
+    kubectl apply -f openebs-storageclasses.yaml
+
+
+Check whether the deployments are running successfully using the following commands.
+
+::
+  
+    name@Master:~$ kubectl get deployments
+    NAME                                            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    maya-apiserver                                  1         1         1            1           2m
+    openebs-provisioner                             1         1         1            1           2m
+  
+::
+  
+    name@Master:~$ kubectl get pod
+    NAME                                   READY     STATUS    RESTARTS   AGE
+    maya-apiserver-1633167387-5ss2w        1/1       Running   0          24s
+    openebs-provisioner-1174174075-f2ss6   1/1       Running   0          23s
+
+
+Check whether the storage classes are applied successfully using the following commands.
+
+::
+  
+    name@Master:~$ kubectl get sc
+    NAME              TYPE
+    openebs-basic     openebs.io/provisioner-iscsi
+    openebs-jupyter   openebs.io/provisioner-iscsi
+    openebs-percona   openebs.io/provisioner-iscsi
+
+    
+Run jupyter-mysql Pod with OpenEBS Storage
+------------------------------------------
+Use OpenEBS as persistent storage for the jupyter pod by selecting an OpenEBS storage class in the persistent volume claim. A sample jupyter pod yaml (with container attributes and pvc details) is available in the OpenEBS git repository (which was cloned in the previous steps).
+::
+   name@Master:~$ cat demo-jupyter-openebs.yaml
+
+   ...
+
+   kind: PersistentVolumeClaim
+   apiVersion: v1
+   metadata:
+     name: jupyter-data-vol-claim
+   spec:
+     storageClassName: openebs-jupyter
+     accessModes:
+       - ReadWriteOnce
+     resources:
+       requests:
+         storage: 5G
+   ...
+
+Apply the jupyter pod yaml using the following command.
+
+::
+
+   name@Master:~$ kubectl apply -f demo-jupyter-openebs.yaml
+   deployment "jupyter-server" created
+   persistentvolumeclaim "jupyter-data-vol-claim" created
+   service "jupyter-service" created
+
+The above command creates the following, which can be verified using the corresponding kubectl commands.
+
+- Launches a Jupyter Server, with the specified notebook file from github (kubectl get deployments)
+- Creates an OpenEBS Volume and mounts to the Jupyter Server Pod (/mnt/data) (kubectl get pvc) (kubectl get pv) (kubectl get pods)
+- Exposes the Jupyter Server to external world via the http://<NodeIP>:32424 (NodeIP is any of the minion nodes external IP) (kubectl get pods)   
+
+The jupyter server dashboard can be accessed on the Kubernetes node port as in the following screen.
```

Comment:
Let us move this step to access jupyter server dashboard after confirming that the pods are running, i.e., the next step in this doc

Reply:
Done!